doc/manual/performance-tips.rst

```diff
@@ -962,7 +962,55 @@ properties:
 -  The stride should be unit stride.
 -  In some simple cases, for example with 2-3 arrays accessed in a loop, the
    LLVM auto-vectorization may kick in automatically, leading to no further
-   speedup with :obj:`@simd`.
+   speedup with :obj:`@simd``.
+
+While :obj:`@simd` needs to be placed in front of a loop, that is not the case 
+for :obj:`@inbounds`. It can be placed in front of any expression as long as it 
+is inside a function, the only place where it could be outside of a function 
+definition is in front of a loop.
+
+    julia> x = [1]
+
+    julia> @inbounds if true
+               x[2]
+           end
+    ERROR: BoundsError: attempt to access 1-element Array{Int64,1} at index [2]
+     in getindex(::Array{Int64,1}, ::Int64) at ./array.jl:309
+     in eval(::Module, ::Any) at ./boot.jl:225
+     in macro expansion at ./REPL.jl:92 [inlined]
+     in (::Base.REPL.##1#2{Base.REPL.REPLBackend})() at ./event.jl:46
+
+    julia> function f(x)
+               @inbounds if true
+                   x[2]
+               end
+           end
+
+    julia> f(x)
+ 
+    julia> @inbounds for i in 1:10
+               x[2]
+           end
+
+Beware using it before a function declaration will have no effect, the 
```

Comment:
Beware -> Be aware that.   Start a new sentence, and rephrase "To use `@inbounds` on a block of code, put it before a `begin ... end` statement.